### PR TITLE
Room template

### PR DESCRIPTION
**Pull request by caseyyee**
image

### CODE REVIEW

**Comment by cvan** [comment]
this looks absolutely fantastic 🎉  I'm loving it 🍟

File: templates/index.json [hunk 1]
```diff
@@ -21,6 +21,21 @@
         "set-image": "./app/js/components/set-image.js"
       },
       "technologies": "ES6"
+    },
+    {
+      "alias": "room-template",
+      "url": "aframevr-userland/room-template",
+      "title": "Room template",
+      "description": "Room built using A-Frame.",
+      "components": {
```
Comment (cvan) [inline templates/index.json]: it doesn't look like any components are used. can clear these out.

File: templates/index.json [hunk 2]
```diff
@@ -21,6 +21,21 @@
         "set-image": "./app/js/components/set-image.js"
       },
       "technologies": "ES6"
+    },
+    {
+      "alias": "room-template",
+      "url": "aframevr-userland/room-template",
```
Comment (cvan) [inline templates/index.json]: this should be `aframevr-userland/aframe-room-template`

File: templates/index.json [hunk 3]
```diff
@@ -21,6 +21,21 @@
         "set-image": "./app/js/components/set-image.js"
       },
       "technologies": "ES6"
+    },
+    {
+      "alias": "room-template",
```
Comment (cvan) [inline templates/index.json]: just call it `room`

File: templates/aframe-room-template/package.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "aframe-room-template",
+  "description": "Room built using A-Frame",
```
Comment (cvan) [inline templates/aframe-room-template/package.json]: end in period

File: templates/aframe-room-template/app/app/js/initialize.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (cvan) [inline templates/aframe-room-template/app/app/js/initialize.js]: why is this two directories deep? in the `app/app` directory? accident?

**Review by cvan — CHANGES_REQUESTED** [review]
this is really impressive, especially considering no models!

File: templates/aframe-room-template/app/app/js/initialize.js (hunk 5 above)
Comment (caseyyee) [reply templates/aframe-room-template/app/app/js/initialize.js]: oops.  copy and paste error.

**Comment by cvan** [comment]
awesome, thanks 👍

**Pull request merged**